### DISCUSSION TITLE
Force specify data encryption algorithm for SSTP SoftEtherVPN

### DISCUSSION THREAD

**Discussion by OmlineEditor** [SoftEtherVPN/SoftEtherVPN#2069] [Q&A]
I have a configured and working server SSTP SoftEtherVPN.
Now my client is my router but I have a weak router and it is slow when working with the SSTP protocol.
I can't select the data encryption algorithm on the client in my router. It is issued by the server and is currently set AES256-CBC.
This algorithm is very high-load and puts a lot of CPU load on my router.

how to force the server to specify the data encryption algorithm for SSTP SoftEtherVPN?

I tried to specify in the console:
ServerCipherSet TLS_AES_128_GCM_SHA256
or
ServerCipherSet RC4-SHA

it doesn't help

- - -
SoftEtherVPN 5.02.5187
Debian 12 x64

**Comment by MitaHill** [comment]
Uh... u can specify **gateway** to another highspeed server running client in your slowy route , if invalid u maybe need replace a better route that

**Comment by MitaHill** [comment]
what's your route running speed ? slowy that cannot solve AES256 **just replace your baby route**HAHA

**Reply by OmlineEditor** [reply]
> what's your route running speed ?

the speed is about 10-20 Megabits per second. I'm not going to change my router. I need to change the encryption algorithm. I understand that there is no way to do this inside the program.

**Reply by chipitsine** [reply]
that's clear. someone will have a look.

**Comment by MitaHill** [comment]
well have good




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "SoftEtherVPN/SoftEtherVPN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月23日(星期三) 凌晨2:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [SoftEtherVPN/SoftEtherVPN] Force specify data encryption algorithm for SSTP SoftEtherVPN (Discussion #2069)





 
that's clear. someone will have a look.

**Comment by MitaHill** [comment]
My about 500 , and&nbsp; I think your router does not support AES that slowy&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "SoftEtherVPN/SoftEtherVPN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月22日(星期二) 晚上11:53
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [SoftEtherVPN/SoftEtherVPN] Force specify data encryption algorithm for SSTP SoftEtherVPN (Discussion #2069)





  
what's your route running speed ?
  
the speed is about 10-20 Megabits per second. I'm not going to change my router. I need to change the encryption algorithm. I understand that there is no way to do this inside the program.
